frontend/src/pages/CourseEditor/TestStep/TestItem/TestItem.tsx

```diff
@@ -14,53 +14,46 @@ import {
   RadioButtonBox,
   RadioSelectAnswer,
   InputAnswer,
-  ButtonBox,
-  QuestionActionButton,
   InputText,
   TestBasicField,
-  InputSelect,
+  FieldSelect,
 } from './styled';
 
 const TestItem: FC<ITestItemProps> = ({ formik, handleChangeAnswer, isCurrentAnswer }) => (
```

Comment:
Please check this one: https://formik.org/docs/examples/radio-group
I suppose, you don't need to use `isCurrentAnswer` useState here